src/test/java/net/snowflake/ingest/SimpleIngestIT.java

```diff
@@ -233,7 +233,7 @@ private void getHistoryAndAssertLoad(SimpleIngestManager manager, String test_fi
     //try to wait until the future is done
     try {
       //wait up to 2 minutes to load
```

Comment:
NIT: Fix comment. Is it taking more than 2 minutes for the load to complete? Should we reduce the number of files?

Reply:
Will fix. We are just adding 1 file, shouldnt take too long. I will revert to 2 mins.